Update silences.md (#38834)
grafana/grafana: docs/sources/alerting/unified-alerting/silences.md

## Create a URL to silence form with defaults filled in

> When linking to silence form, you can provide default matching labels and comment via `matchers` and `comment` query parameters. `matchers` expects one more matching labels of type `[label][operator][value]` joined by a comma. `operator` can be one of `=` (equals, not regex), `!=` (not equals, not regex), `=~` (equals, regex), `!~` (not equals, regex).
> 

Reviewer: ```suggestion
When linking to silence form, provide the default matching labels and comment via `matchers` and `comment` query parameters. The `matchers` parameter require one more matching labels of the type `[label][operator][value]` joined by a comma while the `operator` parameter can be one of  the following: `=` (equals, not regex), `!=` (not equals, not regex), `=~` (equals, regex), `!~` (not equals, regex).

```
